nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -477,7 +477,7 @@ public void commitTable(
     Branch branch =
         getApi()
             .commitMultipleOperations()
-            .operation(Operation.Put.of(key, newTable, expectedContent))
```

Comment:
This change LGTM, but it's a non-trivial change in the Nessie Catalog, certainly not a simple "compiler warning" kind of change... Would you mind moving it into a separate PR for the sake of clarity?